src/app/recruitment/[id]/components/JobDetail.tsx

```
@@ -0,0 +1,66 @@
+        <JobDetailDate
+          expiredDate={job?.expiredDate}
+          uploadDate={job?.uploadDate}
+        />
+        <JobDetailInfo item={job!} />
+        <JobRate />
+      </div>
+      <div className="flex w-full justify-center">
+        {job?.recruitmentContent ? (
+          <div
+            className="w-full rounded bg-white"
+            dangerouslySetInnerHTML={{ __html: job.recruitmentContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)